content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,91 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning">
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
 
 ## Compatibility
+Datadog .NET application performance monitoring supports a robust set of features to provide critical information about .NET applications & services: 
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+- Monitor [runtime metrics][1] to help troubleshoot bugs and detect resource inefficiencies in your application.
 
-- .NET Tracer supports the following processor architectures
-  - Windows x86 (`win-x86`)
-  - Windows x64 (`win-x64`)
-  - Linux x64 (`linux-x64`)
-  - Alpine Linux x64 (`linux-musl-x64`)
-  - Linux ARM64 (`linux-arm64`) Added in version 1.27.0, automatic instrumentation only supported on .NET 5.
+- Ability to correlate [traces & application logs][2].
 
-- The .NET Tracer library for Datadog is open-source. For more information see the [tracer Github repository][1].
+- The .NET Tracer library for Datadog is open-source. For more information, see the [tracer Github repository][3].
 
-<div class="alert alert-warning">
-  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
-</div>
+### .NET runtimes
```

Comment:
Maybe "Supported .NET Runtimes"?

Reply:
Possibly, I thought it would of been inferred.

Reply:
Changed have been updated [here](https://github.com/DataDog/documentation/pull/11200/commits/d4260a023194b2e0791277b5c9242b0903fff21e). 

Marking as resolved.